src/plugins/gui/scenes/FiatPluginWebView.tsx

```diff
@@ -35,10 +37,18 @@ export function FiatPluginWebViewComponent(props: Props): JSX.Element {
     if (onMessage != null) onMessage(data, injectJs)
   })
 
-  React.useEffect(() => () => {
-    // Cleanup code when scene unmounts
-    if (onClose != null) onClose()
-  })
+  // React.useEffect(() => () => {
+  //   // Cleanup code when scene unmounts
+  //   if (onClose != null) onClose()
+  // })
```

Comment:
I don't think we need this old code - the `beforeRemove` approach is clearly superior.